fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -664,6 +680,17 @@ class TabsTrayFragment : AppCompatDialogFragment() {
         }
     }
 
+    private fun showInactiveTabsAutoCloseConfirmationSnackbar() {
+        val text = getString(R.string.inactive_tabs_auto_close_message_snackbar)
+        val snackbar = FenixSnackbar.make(
+            view = tabsTrayComposeBinding.root,
+            duration = FenixSnackbar.LENGTH_SHORT,
+            isDisplayedWithBrowserToolbar = true,
+        ).setText(text)
+        snackbar.view.elevation = ELEVATION
+        snackbar.show()
```

Comment:
I'm going to open a ticket to port this to Compose if this PR is approved

Reply:
We used to have a Snackbar implementation that @MatthewTighe did for wallpapers https://github.com/mozilla-mobile/fenix/pull/23196, but I believe it got removed since it wasn't used after further changes to wallpapers as part of MR2. If we were to port this again, I think we should also consider https://bugzilla.mozilla.org/show_bug.cgi?id=1814198. Perhaps utilize the same ticket if it makes sense to, you could reset the assignee since I am not actively working on it, but just assigned myself due to my general interest in it and to help me easily find it. 

My understanding of snackbars is that it is shown at the app level so if the tab trays fragment got dismissed, the snackbar should still be displayed for the set duration. That said I think introducing a composable snackbar does require some more thoughts. It might make sense to keep using `FenixSnackbar` in the meantime until we have a comprehensive understanding of things

Reply:
Appreciate your thoughts here. I'll hold off for now then.